Source/WebGPU/WebGPU/HardwareCapabilities.h

```diff
@@ -39,6 +39,7 @@ struct HardwareCapabilities {
         bool supportsNonPrivateDepthStencilTextures { false };
         id<MTLCounterSet> timestampCounterSet { nil };
         id<MTLCounterSet> statisticCounterSet { nil };
+        bool canPresent10A2PixelFormats { false };
```

Comment:
What does `10A2` mean in this context? Is it BGR10A2 or RGB10A2 or both?

Reply:
Both, as per https://developer.apple.com/documentation/quartzcore/cametallayer/1478155-pixelformat?language=objc

Reply:
Can we clarify this to mean RGB10A2?

Other devices can present to BGR10A2 or similar 10bits per color formats